main/src/ca/uwaterloo/flix/language/phase/LoadBytecode.scala

```diff
@@ -159,13 +167,15 @@ object LoadBytecode extends Phase[ExecutableAst.Root, ExecutableAst.Root] {
   private def dump(qualName: QualName, code: Array[Byte]): Unit = Files.write(Paths.get(qualName.ref.mkString("", "$", ".class")), code)
 
   /**
+    * TODO: REMOVE THIS.
```

Comment:
???

Reply:
Just a reminder for me to remove this function at some point.